bot/exts/code_jams/_cog.py

```diff
@@ -222,6 +222,19 @@ async def unpin(self, ctx: commands.Context, message: discord.Message | None = N
         """Lets Code Jam Participants to unpin messages in their team channels."""
         await pin_flow(ctx, PIN_ALLOWED_ROLES, self.bot.code_jam_mgmt_api, message, True)
 
+    @codejam.command("ping")
+    @commands.has_any_role(Roles.admins, Roles.events_lead, Roles.code_jam_event_team, Roles.code_jam_participants)
+    @in_code_jam_category(_creation_utils.CATEGORY_NAME)
+    async def ping_codejam_team(self, ctx: commands.Context) -> None:
+        """Ping the team role for the channel this command is ran in."""
+        team_resp = await self.bot.code_jam_mgmt_api.get("/teams/find", params={"name": ctx.channel.name})
```

Comment:
```suggestion
        team_resp = await self.bot.code_jam_mgmt_api.get("teams/find", params={"name": ctx.channel.name})
```

The API seems to 404 because of this extra slash...

Reply:
Resolved in the forcepush https://github.com/python-discord/sir-robin/compare/e42b09c0a8dfbb6a397ecfbc9201e663b66b0bc5..0169ca74bd9e1151f2b5b122cdf67287131c01bf